doc/commands/group.8.rst

```diff
@@ -34,7 +34,8 @@ Description
 ===========
 
 The ``group`` command in ``DNF5`` offers several queries for getting information about groups, packages
-related to them and it is also used for groups installation.
+related to them and it is also used for groups installation. Note that some
+things dnf-4 listed as groups are now listed with ``environment list``.
```

Comment:
I find this new sentence little bit cryptic.

What about to move it to a separate line and change it to something like:
```
To query environments use separate ``environment`` command.
```